Resources/Private/ExtensionArtifacts/composer.json

```diff
@@ -53,7 +53,7 @@
     },
     "require": {
         "helhum/typo3-console": "@dev",
-        "typo3/cms-core": "^10.4.1 || ~11.3.1"
+        "typo3/cms-core": "^10.4.22 || ~11.5.5"
```

Comment:
This for sure must not be changed like that. Rather remove 11 completely like you suggested